crates/propolis-api-types/src/lib.rs

```diff
@@ -77,15 +77,16 @@ pub struct InstanceMigrateStartRequest {
     pub migration_id: Uuid,
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
-pub struct InstanceMigrateStatusRequest {
-    pub migration_id: Uuid,
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq)]
+pub struct InstanceMigrationStatus {
+    pub id: Uuid,
+    pub state: MigrationState,
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq)]
 pub struct InstanceMigrateStatusResponse {
-    pub migration_id: Uuid,
-    pub state: MigrationState,
+    pub migration_in: Option<InstanceMigrationStatus>,
+    pub migration_out: Option<InstanceMigrationStatus>,
 }
```

Comment:
NTS: These structs need doc comments (in particular it should be made clearer that it's possible for a server to report both a prior migration in and a current migration out, and that migration statuses don't get cleared by the conclusion of a migration).